Tests/CoreTests/StrandTests.swift

```diff
@@ -15,18 +15,36 @@ class StrandTests: XCTestCase {
     ]
 
     func testStrand() throws {
-        try (1...10).forEach { _ in
-            var collection = [Int]()
+        try (1...5).forEach { _ in
+            var ran = false
+            let t = try Strand {
+                sleep(1)
+                ran = true
+            }
+            XCTAssertFalse(ran)
```

Comment:
No way to guarantee this will work but if the tests are passing that's fine I guess. >_<